docs/docs/explanations/architecture.md

```diff
@@ -4,7 +4,7 @@ sidebar_label: Architecture
 sidebar_position: 2
 ---
 
-![Kurtosis Architecture](../../static/img/explanations/kurtosis-architecture.png)
+![Kurtosis Architecture](/static/img/explanations/kurtosis-architecture.png)
```

Comment:
this doesn't work

use the @site/static syntax

Reply:
Yeah, using `/img/...` seem to work with both docusaurus versioned docs, and. the linter. So I'll try this